### ISSUE TITLE
Support creating JIRA tickets with the findings

### ISSUE THREAD

**Issue by coheigea** [ossf/allstar#83]
It would be great if AllStar supported creating JIRA tickets instead of just GitHub issues, as many projects use JIRA instead.

I might find time to contribute this next year, if no-one else jumps in before then.

**Comment by jeffmendoza** [comment]
Yes, this sounds great. We don't have a design for Allstar to be able to access secrets at the moment, so I assume this could work if JIRA was able to accept anonymous issues.
